packages/labs/cli/src/lib/generate/generate.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Analyzer} from '@lit-labs/analyzer';
+import {AbsolutePath} from '@lit-labs/analyzer/lib/paths.js';
+
+const frameworkGenerators = {
+  react: () => import('../generate/react.js'),
+};
+
+type FrameworkName = keyof typeof frameworkGenerators;
+
+export const run = async (
+  {packageRoot, frameworks}: {packageRoot: string; frameworks: string[]},
+  console: Console
+) => {
+  const analyzer = new Analyzer(packageRoot as AbsolutePath);
+  const analysis = analyzer.analyzePackage();
+  await Promise.all(
```

Comment:
I think you want [`Promise.allSettled`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) here.

Reply:
Done